extension/encoding/googlecloudlogentryencodingextension/log_proto_payload_test.go

```diff
@@ -194,10 +192,6 @@ func TestProtoErrors(t *testing.T) {
   }
 }`),
 			expectsErr: "google.cloud.audit.AuditLog has no known field with JSON name ServiceName",
```

Comment:
How will you handle AuditLog schema evolution? It's not the first time a new field is introduced?

Reply:
Just to give more details, we will do something similar as we do for AWS logs encoding already: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/972d7ae7c300d0c609ba54e41ac2c614a22d0ee0/extension/encoding/awslogsencodingextension/internal/unmarshaler/waf/unmarshaler.go#L34-L61